### DISCUSSION TITLE
Upgrade helper is broken

### DISCUSSION THREAD

**Discussion by szujak** [microsoft/react-native-windows#14125] [General]
Hello, I wanted to update my react-native-windows project from 0.75.7 to 0.76.1 version. As the update is big I checked the upgrade helper (https://react-native-community.github.io/upgrade-helper) but the last visible version there is 0.74.22.

I've checked the source code of that project and it looks like in the background it use https://github.com/acoates-ms/rnw-diff repository to get the diff for all versions, but github actions in this repository are broken since March 2024 and the last update of RELEASES is from October 2024.

Is there any chance to fix that?

**Comment by chrisglein** [comment]
Created an issue to track looking into this: https://github.com/microsoft/react-native-windows/issues/14151
